runner/engine.go

```diff
@@ -239,9 +239,6 @@ func (e *Engine) watchPath(path string) error {
 				return
 			case ev := <-e.watcher.Events():
 				e.mainDebug("event: %+v", ev)
-				if !validEvent(ev) {
```

Comment:
If the check is removed, the file checksum will be broken. I need to keep this logic...

Reply:
what is the consequence of adding `CHMOD` to the list of valid events?